chain/x/oracle/keeper/result.go

```diff
@@ -50,3 +50,30 @@ func (k Keeper) AddResult(
 	store.Set(types.ResultStoreKey(id), resultHash)
```

Comment:
Now that we have `SetResult`. I think this should call into `SetResult`.

Reply:
fixed, used set result function